authority/authority.go

```diff
@@ -117,6 +121,22 @@ func (a *Authority) init() error {
 		}
 	}
 
+	// Decrypt and load SSH keys
+	if a.config.SSH != nil {
+		if a.config.SSH.HostKey != "" {
+			a.sshCAHostCertSignKey, err = parseCryptoSigner(a.config.SSH.HostKey, a.config.Password)
```

Comment:
Should this be the same password as everything else? I assume this is the same password as the intermediate private key uses.

Reply:
That's the current functionality for MVP, but we can change it if we need to.